### PR TITLE
Add template tag to set column class based on number of office locations

### PR DESCRIPTION
**Pull request by davidebukali**
Fix for [#410](https://github.com/devinit/DIwebsite-redesign/issues/410)

### CODE REVIEW

File: di_website/about/templatetags/count_columns.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+
+register = template.Library()
+
+
+@register.simple_tag
+def columns():
+    number_of_offices = OfficeLocation.objects.exclude(
+        latitude__isnull=True).exclude(longitude__isnull=True).all().count()
+    return number_of_offices
```
Comment (akmiller01) [inline di_website/about/templatetags/count_columns.py]: Looks like a good fix @davidebukali but we might want to wait for feedback from @benjamincoleman on what classes are actually meaningful. I'm not sure whether `l-1up` would be valid, and I'm assuming we want to max out at `l-4up`

At which point this line could become: 
```
return max(2, min(number_of_offices, 4))
```
Which would bound it between 2 and 4.

Comment (davidebukali) [reply di_website/about/templatetags/count_columns.py]: Thanks @akmiller01; `return max(2, min(number_of_offices, 4))
` is a neat constraint

Comment (akmiller01) [reply di_website/about/templatetags/count_columns.py]: Just doing a quick search through the pattern library @davidebukali 

Just as I suspected, it starts at 2, but goes to 5. So you could use `max(2, min(number_of_offices, 5))`

Comment (davidebukali) [reply di_website/about/templatetags/count_columns.py]: @akmiller01 this was updated and pushed

**Pull request merged**